library/classes/CouchDB.class.php

```diff
@@ -33,6 +21,7 @@ function __construct()
         $this->user = ($GLOBALS['couchdb_user'] != '') ? $GLOBALS['couchdb_user'] : null;
         $this->pass = ($GLOBALS['couchdb_pass'] != '') ? $GLOBALS['couchdb_pass'] : null;
         $this->port = $GLOBALS['couchdb_port'];
+        $this->ssl_port = $GLOBALS['couchdb_ssl_port'];
```

Comment:
Shouldn't ssl be a binary choice ('http' or 'https') in GLOBAL 'couchdb_protocol'?  Additional port setting shouldn't be needed.
[Apache Docs](http://docs.couchdb.org/en/2.1.1/config/http.html#secure-socket-level-options)
[Confluence Docs](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=48203146)

Also it would greatly help to bring in [PHP on Couch](https://github.com/dready92/PHP-on-Couch) or something similar.